cmd/goordinator/main.go

```diff
@@ -280,6 +287,65 @@ func hide(in string) string {
 	return "**hidden**"
 }
 
+func startPullRequestAutoupdater(config *cfg.Config, githubClient *githubclt.Client, mux *http.ServeMux) (*autoupdate.Autoupdater, chan<- *github.Event) {
+	if !config.Autoupdater.TriggerOnAutoMerge && len(config.Autoupdater.Labels) == 0 {
+		logger.Info(
+			"github pull-request updater is disabled, trigger_on_auto_merge is false and trigger_labels in config is empty",
+			logfields.Event("autoupdater_disabled"),
+		)
+
+		return nil, nil
+	}
+
+	if len(config.GithubAPIToken) == 0 {
+		fmt.Fprintf(os.Stderr, "ERROR: config file %s: github_api_token must be provided when autoupdate is enabled", *args.ConfigFile)
```

Comment:
I might have gotten this wrong from the very beginning but... is it autoupdate or autoupdater?
Is the module name autoupdater but the feature autoupdate?

Reply:
the component is called autoupdater, so it should here also be called "autoupdater", I'll change it, thx